### ISSUE TITLE
pretrained: False

### ISSUE THREAD

**Issue by seungbo** [ultralytics/ultralytics#17246]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I fixed the “[default.yaml](https://github.com/ultralytics/ultralytics/blob/main/ultralytics/cfg/default.yaml)” file to “pretrained: False”, but why does the yolo11n.pt file download automatically?
I don't want to use `pretrained mode`.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @seungbo, thank you for reaching out and your interest in Ultralytics 🚀! This is an automated response, but rest assured that an Ultralytics engineer will assist you soon.

We recommend checking out our [Docs](https://docs.ultralytics.com) where you can find detailed information about configurations and settings. For configuring and customizing model settings, you might find our [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples particularly helpful.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us debug it more effectively.

For those interested in custom training or troubleshooting, ensure you provide detailed information, such as dataset image examples and relevant training logs, and refer to our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join our community for more support and discussion! Engage in real-time chat on [Discord](https://ultralytics.com/discord) 🎧, explore in-depth topics on [Discourse](https://community.ultralytics.com), or dive into community threads on our [Subreddit](https://reddit.com/r/ultralytics).

## Upgrade

Make sure you are using the latest version of the `ultralytics` package. Upgrade with:

```bash
pip install -U ultralytics
```

Ensure your environment meets all the [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) and is running on [**Python>=3.8**](https://www.python.org/) and [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/).

## Environments

Feel free to explore different environments where YOLO can be executed with all dependencies preinstalled:

- **Notebooks** with GPU support: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

The current status of our CI tests can help verify if everything is functioning as expected. If this badge is green, all tests are passing for our supported environments.

Your patience and cooperation are greatly appreciated, and an engineer will provide further assistance shortly. 😊

**Comment by seungbo** [comment]
Why don't `pretrained` arguments work?

**Comment by Y-T-G** [comment]
`pretrained` argument is used to pass a path to a model to load weights from like `pretrained=yolo11n.pt`. Setting it `True` or `False` will not have an effect unless it's classification. If you want to train from scratch, you should be using `.yaml` as the model `yolo11n.yaml`.

**Comment by seungbo** [comment]
Why did I set the argument “epochs=60” but it changed to 100?

**Comment by Y-T-G** [comment]
What's the full code?

**Comment by seungbo** [comment]
`!yolo detect train epochs=60 imgsz=640 model=/content/drive/MyDrive/seungbo/yolo11n.yaml cfg=/content/drive/MyDrive/seungbo/hyp.yaml`

yolo11n.yaml
"
mixup: 0.35

optimizer: SGD
workers: 32
batch: 256
erasing: 4.5
mixup: 0.3
epochs: 60

mode: "train"
model: "/content/drive/MyDrive/seungbo/yolo11n.yaml"
data: '/content/drive/MyDrive/seungbo/data.yaml'
"

yolo11n.yaml
"
nc: 9 
scales:
  n: [0.50, 0.25, 1024] # summary: 319 layers, 2624080 parameters, 2624064 gradients, 6.6 GFLOPs
  s: [0.50, 0.50, 1024] # summary: 319 layers, 9458752 parameters, 9458736 gradients, 21.7 GFLOPs
  m: [0.50, 1.00, 512] # summary: 409 layers, 20114688 parameters, 20114672 gradients, 68.5 GFLOPs
  l: [1.00, 1.00, 512] # summary: 631 layers, 25372160 parameters, 25372144 gradients, 87.6 GFLOPs
  x: [1.00, 1.50, 512] # summary: 631 layers, 56966176 parameters, 56966160 gradients, 196.0 GFLOPs

backbone:
  - [-1, 1, Conv, [64, 3, 2]] # 0-P1/2
  - [-1, 1, Conv, [128, 3, 2]] # 1-P2/4
  - [-1, 2, C3k2, [256, False, 0.25]]
  - [-1, 1, Conv, [256, 3, 2]] # 3-P3/8
  - [-1, 2, C3k2, [512, False, 0.25]]
  - [-1, 1, Conv, [512, 3, 2]] # 5-P4/16
  - [-1, 2, C3k2, [512, True]]
  - [-1, 1, Conv, [1024, 3, 2]] # 7-P5/32
  - [-1, 2, C3k2, [1024, True]]
  - [-1, 1, SPPF, [1024, 5]] # 9
  - [-1, 2, C2PSA, [1024]] # 10

head:
  - [-1, 1, nn.Upsample, [None, 2, "nearest"]]
  - [[-1, 6], 1, Concat, [1]] # cat backbone P4
  - [-1, 2, C3k2, [512, False]] # 13

  - [-1, 1, nn.Upsample, [None, 2, "nearest"]]
  - [[-1, 4], 1, Concat, [1]] # cat backbone P3
  - [-1, 2, C3k2, [256, False]] # 16 (P3/8-small)

  - [-1, 1, Conv, [256, 3, 2]]
  - [[-1, 13], 1, Concat, [1]] # cat head P4
  - [-1, 2, C3k2, [512, False]] # 19 (P4/16-medium)

  - [-1, 1, Conv, [512, 3, 2]]
  - [[-1, 10], 1, Concat, [1]] # cat head P5
  - [-1, 2, C3k2, [1024, True]] # 22 (P5/32-large)

  - [[16, 19, 22], 1, Detect, [nc]] # Detect(P3, P4, P5)
"

data.yaml
"
path: /content/drive/MyDrive/seungbo/datasets/
train: images/train
val: images/val

names:
    0: spring fish trap
    1: rope
    2: tire
    3: fish net
    4: circular fish trap
    5: wood
    6: rectangular fish trap
    7: eel fish trap
    8: bundle of rope
"

**Comment by Y-T-G** [comment]
Why are you passing `cfg`?

**Comment by seungbo** [comment]
It looks like this: `cfg=/content/drive/MyDrive/seungbo/hyp.yaml`.

**Comment by Y-T-G** [comment]
You should either only use cfg or manually pass args. Not both. And you should have the complete `cfg`, not only a few arguments.

https://github.com/ultralytics/ultralytics/blob/main/ultralytics/cfg/default.yaml
